pootle/apps/pootle_project/models.py

```diff
@@ -56,6 +59,20 @@
 
 class ProjectManager(models.Manager):
 
+    def create(self, *args, **kwargs):
```

Comment:
k - i experimented with this. My feeling that in terms of the CRUD cycle the manager is the right place, and has the advantage of making it easier to create a project passing a list of string names for `filetypes`